orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -45,6 +45,7 @@ struct TrainingParameters {
   int horizontal_parallel_size = 1;
   int deepspeed_zero_stage = 0;
   bool enable_grad_norm_clip = true;
+  bool enable_gelu_approximation = false;
```

Comment:
>enable_gelu_approximation [](start = 7, length = 25)

Could you add a test case for the new option in orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py?

Reply:
Yes. I will make a test to check if saved ONNX model contains FastGelu node on saving.